utils/transformers/index.js

```diff
@@ -30,3 +31,240 @@ export const transformStyle = ({ nodeName, nodeValue, fillProp }) => {
 
 export const getEnabledAttributes = enabledAttributes => ({ nodeName }) =>
   enabledAttributes.includes(dashToCamelCase(nodeName));
+
+const COMMON_ATTS = [
+  'id',
+  'fill',
+  'fillOpacity',
+  'stroke',
+  'strokeWidth',
+  'strokeOpacity',
+  'opacity',
+  'strokeLinecap',
+  'strokeLinejoin',
+  'strokeDasharray',
+  'strokeDashoffset',
+  'x',
+  'y',
+  'rotate',
+  'scale',
+  'origin',
+  'originX',
+  'originY',
+  'transform',
+  'clipPath',
+  'fillRule',
+];
+
+export const obtainComponentAtts = (
+  { attributes },
+  enabledAttributes,
+  fill,
+  fillAll,
+) => {
+  const styleAtts = {};
+  const attributeItems = Array.from(attributes);
+
+  if (fill && fillAll) {
```

Comment:
I know it came from the current code, but do you know why `fillAll` is necessary?

Reply:
By using only the `fill`, it will replace every `fill` attribute that hasn't `"none"` as value to `fill`'s value and keep all `"none"` fills infact. If `fillAll` is `true`, it will do the same as before for **all** `fill` attributes.

Reply:
got it! 👍